application.windows64/java/Welcome.html

```
@@ -0,0 +1,28 @@
+<html>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)